### PR TITLE
Add auto verify to config

### PR DESCRIPTION
**Pull request by desoindx**

### CODE REVIEW

File: .env.ci [hunk 1]
```diff
@@ -26,6 +26,7 @@ FEATURE_REIMBURSEMENT_PAGE=true
 
 # Activate the auto accept feature on DS
 FEATURE_AUTO_ACCEPT=true
+FEATURE_AUTO_VERIFY=true
```
Comment (sandrica89) [inline .env.ci:29]: No need it's just below

File: utils/config.js [hunk 2]
```diff
@@ -38,6 +38,7 @@ module.exports = {
   featureSendSummary: process.env.FEATURE_SEND_SUMMARY || false,
   featurePsyPages: process.env.FEATURE_PSY_PAGES || false,
   featureAutoAccept: process.env.FEATURE_AUTO_ACCEPT || false,
+  featureAutoVerify: process.env.FEATURE_AUTO_VERIFY || false,
```
Comment (sandrica89) [inline utils/config.js:41]: Sorry I forgot this one in previous PR :(

**Review by sandrica89 — CHANGES_REQUESTED** [review]
Some new code can be removed

**Comment by desoindx** [comment]
> Some new code can be removed

sorry !
c'est pas facile ce matin :p

**Review by sandrica89 — APPROVED** [review]

**Pull request merged**